libraries/SITL/SIM_Aircraft.h

```diff
@@ -178,6 +185,7 @@ class Aircraft {
     const char *frame;
     bool use_time_sync = true;
     float last_speedup = -1.0f;
+    std::string config_ = "";
```

Comment:
we'd prefer to use char* instead

Reply:
fixed, all std::strings changed to use char*.